addons/hr/views/hr_contract_type_views.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <record id="action_hr_contract_type" model="ir.actions.act_window">
+            <field name="name">Job Positions</field>
+            <field name="res_model">hr.job</field>
+            <field name="view_mode">tree</field>
+        </record>
+
+        <record id="hr_contract_type_view_tree" model="ir.ui.view">
+            <field name="model">hr.contract.type</field>
+            <field name="arch" type="xml">
+                <tree string="Contract Types" editable="bottom">
+                    <field name="sequence" widget="handle"/>
+                    <field name="name"/>
+                </tree>
+            </field>
+        </record>
+
+        <record id="hr_contract_type_view_form" model="ir.ui.view">
+            <field name="model">hr.contract.type</field>
+            <field name="arch" type="xml">
+                <form>
+                    <group>
+                        <group>
+                            <field name="sequence" />
+                            <field name="name" />
+                        </group>
+                    </group>
+                </form>
+            </field>
+        </record>
```

Comment:
Is the form definition needed if the records are editable in the tree view and the view mode of the action window is only tree? Asking for a friend
Edit: My friend tried and has seen no issues when removing the form definition